sdk/textanalytics/Azure.AI.TextAnalytics/assets.json

```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "net",
   "TagPrefix": "net/textanalytics/Azure.AI.TextAnalytics",
-  "Tag": "net/textanalytics/Azure.AI.TextAnalytics_b4a81c595d"
+  "Tag": "net/textanalytics/Azure.AI.TextAnalytics_28b9ae88a5"
```

Comment:
Correct, I didn't re-record existing tests. But I did record a few new tests because, after I migrated all the existing tests to use the new LRO methods, I also wanted to keep some test collateral for the old LRO methods too to guard against breaking changes.